joomla/joomla-cms
# [4.0] Use the latest polyfills for web components

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/20331.

### Summary of Changes
Version 2 of the polyfills:
- improved loader
- no more html imports

### Testing Instructions
Go to admin area edit an article and check that:
- the switcher
- the media field
- the user field
and the alerts working as expected

(tests in EDGE and IE11 will be nice here) 

### Polyfills loaded:
![screen shot 2018-05-16 at 16 53 48](https://user-images.githubusercontent.com/3889375/40124904-da7846ae-5929-11e8-814c-a188540e0089.png)




### Documentation Changes Required

@wilsonge
